riocli/configtree/tree.py

```diff
@@ -48,7 +49,7 @@ def create_config_tree(
     config = get_config_from_context(ctx)
 
     try:
-        client = config.new_v2_client(with_project=with_project)
+        client = config.new_v2_client(with_project=with_org)
```

Comment:
todo: I think this will also be (not with_org)

Reply:
Done.